spec/fixtures/files/supervisor_volunteers.csv

```diff
@@ -1,3 +1,3 @@
-﻿email,display_name,supervisor_volunteers,phone_number
-s5@example.com,s5,volunteer1@example.net,11111111111
-s6@example.com,s6,volunteer1@example.net,11111111111
+﻿email,display_name,supervisor_volunteers,phone_number
+s5@example.com,s5,volunteer1@example.net,11111111111
+s6@example.com,s6,volunteer1@example.net,11111111111
```

Comment:
The diffs in the files here are from a CRLF -> LF change to appease `git mv`. No actual changes to any of the .csvs.